frontend/src/app/components/wp-table/configuration-modal/tabs/display-settings-tab.component.ts

```diff
@@ -26,14 +26,14 @@ export class WpTableConfigurationDisplaySettingsTab implements TabComponent {
     title: this.I18n.t('js.label_group_by'),
     placeholder: this.I18n.t('js.placeholders.default'),
     please_select: this.I18n.t('js.placeholders.selection'),
+    default: '— ' + this.I18n.t('js.work_packages.table_configuration.default'),
     display_sums: this.I18n.t('js.work_packages.query.display_sums'),
-    display_sums_hint: this.I18n.t('js.work_packages.table_configuration.display_sums_hint'),
+    display_sums_hint: '- ' + this.I18n.t('js.work_packages.table_configuration.display_sums_hint'),
```

Comment:
There are different kinds of dashes for the same kind of hints? Can you make this the longer em-dash as well?

Reply:
Wow, I am impressed that you found that. Actually in my IDE you can hardly see a difference. However in the Browser it becomes obvious indeed.